lib/winnow/model.rb

```diff
@@ -83,12 +83,20 @@ def fts_scope_for(column)
         "(match(#{table_name}.#{column}) against(? in boolean mode) and (#{table_name}.#{column} like ?))"
       end
 
-      def fts_tokens_for(term)
+      SPECIAL_CHARS = %r{[@~"<>{}()+*\-]+}
+
+      def fts_starts_with_tokens_for(term)
         # since we're searching in boolean mode, strip out search operators and tokenize.
-        tokens = term.gsub(%r{[@~"<>{}()+*\-]+}, '* ')
+        tokens = term.gsub(SPECIAL_CHARS, '* +')
         tokens = "#{tokens}*".sub(%r{\* +\*+$}, '*')
```

Comment:
Are we missing a `+` in front of the first term here?

Reply:
Yeah, I had misread the docs. They were saying no '+*' not no leading plusses entirely.